docs/docs/contribute/docs/write-documentation.md

```diff
@@ -0,0 +1,270 @@
+
+        ![Using ... to automatically list pages.](assets/editing-ref-pages-dots.png)
+
+## Generating Documentation for the Shards API
+
+You might have observed that the `/docs/docs/reference/shards` folder is conspicuously empty. This is to allow for the faster previewing of your static site without having to load all the pages for the Shards API.
+
+To generate the documentation for the Shards API:
+
+1. Launch your MinGW terminal.
+
+2. Navigate to where your Shards repository is located using the command `cd $(cygpath -u '(X)')`, where (X) is the directory of your folder.
+
+        cd $(cygpath -u '(X)')
+
+    If your repository is located at `C:\Fragcolor\Shards`, the command used in the MinGW terminal would be `cd $(cygpath -u 'C:\Fragcolor\Shards')`.
+
+ 3. Input the following command:
+
+        ./build/Debug/shards ./docs/generate.edn
+
+!!! note
+    The **Debug** version of Shards must be used for the generating of the Shards API pages.
+
+Once done, you will notice that the directory is now filled with folders and Markdown files of various shards.
+
+## Generating Log Files for Examples
+
+Did you realize that the "Output" segments for your code examples are empty too? This is due to how a `.log` file is required for each example to show its output.
+
+To generate the `.log` files for your code examples:
+
+1. Launch your MinGW terminal.
+
+2. Navigate to where your Shards repository is located using the command `cd $(cygpath -u '(X)')`, where (X) is the directory of your folder.
+
+        cd $(cygpath -u '(X)')
+
+    If your repository is located at `C:\Fragcolor\Shards`, the command used in the MinGW terminal would be `cd $(cygpath -u 'C:\Fragcolor\Shards')`.
+
+ 3. Input the following command:
+
+        ./run-samples.sh --release
+
+!!! warning "Segmentation fault"
+    If the error for "Segmentation fault" appears, run the command below before attempting Step 3 again.
+
+        export GFX_BACKEND=D3D12
+
+When the script is done, you will notice new `.log` files within your "Samples" folder. These will be shown in your documentation as the "Output" of your code examples.
+
+![The results with the .edn.log file.](assets/editing-examples-log-md.png)
+
+![The log file will populate the "Output" segment.](assets/editing-examples-site-output.png)
+
+## Updating the Shards API Docs
+
+As you have seen from the segment on [navigating the docs](#navigating-the-docs), a shard's API documentation is generated from three different locations:
+
+1. `/docs/details` folder
+
+2. `/docs/samples` folder
+
+3. `/docs/docs/reference` folder
+
+Each segment will require a different approach for making changes. The following are the steps required to work with each segment:
+
+### Details
+
+1. Navigate to the `/docs/details` folder.
+
+2. Search for the .md file with the shard's name. We will be using the `Msg` shard in this example.
```

Comment:
File could be missing. We should explain how to add a details for a shard that is not already listed here.

i.e. create a folder for the category if it doesn't exist yet and a file named with the shard's name and ending with .md extension.

Reply:
The context currently would be giving instructions on how to amend existing files. I will add a note to create the file if it does not exist.

```
!!! note
        If the file does not exist, create a new `.md` file with the shard's name (e.g., `Msg.md`).
```